test/table.ts

```diff
@@ -1109,350 +1125,164 @@ describe('Bigtable/Table', () => {
       });
     });
 
-    describe('retries', () => {
```

Comment:
These tests used to mock out the request function. They have now been moved to read-rows-retry-test.json.